### DISCUSSION TITLE
Image color is changed when print

### DISCUSSION THREAD

**Discussion by Georgeqhh** [nroduit/Weasis#393] [Q&A] [answered]
When I print the image in my web app,the image color is changed,but print in weasis is OK,someone can help me find the reason and resolve it?
my web app code:
![image](https://user-images.githubusercontent.com/16661018/233246748-428da39a-fa10-43f0-ba9d-bd169b167394.png)

![image](https://user-images.githubusercontent.com/16661018/233244586-f8269035-f4af-44f9-99a2-e4d090efa09f.png)
my web app:
![image](https://user-images.githubusercontent.com/16661018/233244159-df3059d9-37d3-4f4e-b571-e46ae0e3c16b.png)
weasis:
![image](https://user-images.githubusercontent.com/16661018/233244311-f3b5ff4a-d189-48cd-b5d7-e7d14a3f41f0.png)
weasis print setting:
![image](https://user-images.githubusercontent.com/16661018/233246301-5737b536-6a6d-4446-ab21-d42d6b1bfda3.png)

**Comment by Georgeqhh** [comment]
I find the image type is 5,weasis is 0,so how can I get the right type image?

my web app:
![image](https://user-images.githubusercontent.com/16661018/233249123-a7aab9cf-c263-4704-aac9-730fa8604dbb.png)
![image](https://user-images.githubusercontent.com/16661018/233249258-7f2d9ae7-7ade-4025-baf7-cf4486bf67bf.png)

weasis:
![image](https://user-images.githubusercontent.com/16661018/233249324-dd0051b6-d1e2-41d6-bb10-1cbc5e30c2f7.png)
![image](https://user-images.githubusercontent.com/16661018/233249608-ba197dc1-8cd0-4bf9-b2da-4b91574ad4b0.png)

**Comment by nroduit** [comment]
As it is already difficult for me to find the time to answer requests for the projects I contribute to, I cannot assist in configuring or debugging other applications.

**Reply by Georgeqhh** [reply, marked as answer]
Thank you for your reply,I have resolved it.Draw a rgb empty image,then draw my image into it.
